### PR TITLE
update Identity versions April 2020

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

**Comment by g2vinay** [comment]
@mitchdenny 
What is the java - parents - ci below?
It is failing for unknown reason.

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -53,3 +53,4 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # beta_<groupId>:<artifactId>;dependency-version
 # note: Released beta versions will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
+beta_com.azure:azure-identity;1.1.0-beta.3
```
Comment (alzimmermsft) [inline eng/versioning/version_client.txt:56]: I think this should be removed since nothing is using it yet, don't want to add these eagerly.

Comment (g2vinay) [reply eng/versioning/version_client.txt]: From the description of this feature.
This is needed for Identity as we don't want post release automated PRs to change dep version of Identity with beta versions.
But, its best to have @JimSuplizio  review this, so will add it via separate PR.

Comment (g2vinay) [reply eng/versioning/version_client.txt]: Removed for now.

**Comment by mitchdenny** [comment]
> @mitchdenny
> What is the java - parents - ci below?
> It is failing for unknown reason.

Parents is a new pipeline going in to support parent POM publishing via unified publishing. Still a WIP.

**Pull request merged**